code/__DEFINES/is_helpers.dm

```diff
@@ -172,7 +172,7 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 
 #define ismecha(A) (istype(A, /obj/mecha))
 
-#define is_cleanable(A) (istype(A, /obj/effect/decal/cleanable) || istype(A, /obj/effect/rune)) //if something is cleanable
+#define is_cleanable(A) ((istype(A, /obj/effect/decal/cleanable) || istype(A, /obj/effect/rune)) && !istype(A, /obj/effect/decal/cleanable/crayon/gang)) //if something is cleanable
```

Comment:
why? what exactly is preventing gang tags from being considered cleanable?

Reply:
for some reason they just weren't getting hit by cleaning stuff that wasn't Soap, i couldn't find any runtimes related to it and they still call ..() in their Destroy, this seemed to work

Reply:
I would really appreciate if you were to look at the source of this instead of adding further tech debt.

Reply:
its on the todo list for sure, but i've been busy trying to get the higher priority bugs fixed for now, this will be properly looked at before merging